lib/resty/ldap/client.lua

```diff
@@ -16,6 +18,15 @@ local _M = {}
 local mt = { __index = _M }
 
 
+local function to_hex_sequence(data)
```

Comment:
We can use `to_hex` in https://github.com/openresty/lua-resty-string/

Reply:
moved